### PR TITLE
Get add event members (registrations) route

### PR DESCRIPTION
**Pull request by connernilsen**

### CODE REVIEW

File: api/src/main/java/com/codeforcommunity/rest/subrouter/FailureHandler.java [hunk 1]
```diff
@@ -124,6 +125,11 @@ public void handleS3FailedUpload(RoutingContext ctx, String exceptionMessage) {
     end(ctx, message, 502);
   }
 
+  public void handleEventDoesNotExistException(RoutingContext ctx, EventDoesNotExistException exception) {
+     String message = "There does not exist an event with id: " + exception.getEventId();
```
Comment (liammoyn) [inline api/src/main/java/com/codeforcommunity/rest/subrouter/FailureHandler.java:129]: "There does not exist an event" -> "There is no event"

File: service/src/main/java/com/codeforcommunity/processor/EventsProcessorImpl.java [hunk 2]
```diff
@@ -174,6 +180,25 @@ public void deleteEvent(int eventId, JWTData userData) {
     db.delete(EVENTS).where(EVENTS.ID.eq(eventId)).execute();
   }
 
+  @Override
+  public EventRegistrations getEventRegisteredUsers(int eventId, JWTData userData) {
+    if (userData.getPrivilegeLevel() != PrivilegeLevel.ADMIN) {
+      throw new AdminOnlyRouteException();
+    }
+
+    db.selectFrom(EVENTS).where(EVENTS.ID.eq(eventId)).fetchOptional()
```
Comment (liammoyn) [inline service/src/main/java/com/codeforcommunity/processor/EventsProcessorImpl.java:189]: db.fetchExists

File: api/src/main/java/com/codeforcommunity/rest/subrouter/FailureHandler.java (hunk 1 above)
Comment (connernilsen) [reply api/src/main/java/com/codeforcommunity/rest/subrouter/FailureHandler.java]: Fixed

File: service/src/main/java/com/codeforcommunity/processor/EventsProcessorImpl.java (hunk 2 above)
Comment (connernilsen) [reply service/src/main/java/com/codeforcommunity/processor/EventsProcessorImpl.java]: Fixed

**Pull request merged**